doc/filters/country_name.rst

```diff
@@ -16,6 +16,32 @@ By default, the filter uses the current locale. You can pass it explicitly:
     {# États-Unis #}
     {{ 'US'|country_name('fr') }}
 
+The locale can contain more than two letters depending on the region:
+
+.. code-block:: twig
+
+    {# 美國 #}
+    {{ 'US'|country_name('zh_Hant_HK') }}
+
+.. note::
+
+    For more information on the format of the locale:
+
+    See https://www.rfc-editor.org/info/bcp47 for the specifications.
+
+    It is documented by https://www.php.net/manual/en/class.locale.php.
```

Comment:
those URLs should be links.

And I think the explanation should go in the `Arguments` section rather than above the note about which package to install, with something like this: `locale: The locale code as defined in RFC 5646` (with `RFC 5646` being a link to the RFC). Btw, I suggest linking directly to the RFC 5646 instead of this BCP47 thing which groups 2 RFCs (as the RFC 4647 is irrelevant in this context anyway)